r-package/R/download.R

```diff
@@ -120,27 +142,80 @@ download <- function(
 read_sql <- function(
   query,
   billing_project_id = get_billing_id(),
-  page_size = 1000) {
+  page_size = 100000) {
 
   if(billing_project_id == FALSE) {
 
     rlang::abort("You haven't set a Project Billing Id. Use the function `set_billing_id` to do so.")
 
   }
 
-  if(!rlang::is_character(query)) {
+  if(!rlang::is_string(query)) {
 
-    rlang::abort("`query` argument must contain valid SQL text")
+    rlang::abort("`query` argument must be a string.")
 
   }
 
   bigrquery::bq_project_query(
-      billing_project_id,
-      query = query) %>%
+    billing_project_id,
+    query = query) %>%
     bigrquery::bq_table_download(
       page_size = page_size,
       bigint = "integer64")
 
 }
```

Comment:
Aqui não rolou: x Not found: Table basedosdados:br_ibge_populacao.municipios was not found in location US [notFound]

Reply:
@fernandascovino @mavalentim aquele problema lá de novo